node/pkg/logscribeconsumer/app.go

```diff
@@ -75,11 +74,16 @@ func (a *App) Write(p []byte) (n int, err error) {
 		return 0, errorsentinel.ErrLogEmptyLogByte
 	}
 
-	if !a.isLogLevelValid(res) {
+	// check if the log should be sent to logscribe
+	if isLogLevelValid(res, a.Level) {
+		a.buffer <- res
+	}
+
+	// check if the log should be written to the console
+	if !isLogLevelValid(res, DefaultLogConsoleLevel) {
 		return 0, nil
 	}
```

Comment:
```suggestion
	// check if the log should be written to the console
	if !isLogLevelValid(res, DefaultLogConsoleLevel) {
		return 0, nil
	}
	a.buffer <- res
```
as far as I understand, logs will be written in console through 
if you already called zerolog.SetGlobalLevel(), you won't need to make additional post processing

```
_, err = a.consoleWriter.Write(p)
	if err != nil {
		return 0, err
	}
```

Reply:
hmm i see... but shouldn't we still check if the log level is above the logscribe/db level before we write the buffer channel?

Reply:
awh so you put LogConsoleLevel there, I think you can
```
if !isLogLevelValid(res, logStoreLevel) {
		return 0, nil
	}
	a.buffer <- res
```

Reply:
got it, will give it a try!